### PR TITLE
Fix for issue where scoreboards do not update properly

### LINKED ISSUE #23: Player health scoreboard inconsistent
**Describe the bug**
When all players are full health and a new player joins, only their health is shown until other players take damage.

**Expected behavior**
All players' health should be shown at all times.

### PR DESCRIPTION
**Pull request by leviem1**
Resolves #23

### CODE REVIEW

**Comment by leviem1** [comment]
@zupmine We need to test this manually then once we see that it works you should approve it.

**Review by zupmine — APPROVED** [review]
It Works!!

**Pull request merged**
